### ISSUE TITLE
doucment error at https://docs.arangodb.com/3.3/AQL/Operations/Update.html

### ISSUE THREAD

**Issue by dsonet** [arangodb/arangodb#4778]
Start from the section
```javascript
FOR u IN users
  UPDATE u WITH { value: "test" } 
  LET previous = OLD 
  RETURN previous._key
```
all the examples are wrong. They are missing `IN users` before LET. The shell didn't report correct error in this situation instead of the followings:

Query: AQL: syntax error, unexpected RETURN declaration near 'RETURN previous._key' at position 2:1 (while parsing)

**Comment by maxkernbach** [comment]
Hi @dsonet,

thanks a lot for reporting. This should be fixed by #5079, #5080 and #5081.

**Comment by dsonet** [comment]
@maxkernbach Thanks a lot. Hope also can produce more reasonable error message for this situation.
